crates/ark/src/modules/positron/graphics.R

```diff
@@ -222,6 +193,142 @@ with_graphics_device <- function(
     expr
 }
 
+use_ragg <- local({
+    # Only check global option once per session
+    USE_RAGG <- NULL
+
+    function() {
+        if (is.null(USE_RAGG)) {
+            USE_RAGG <<- init_use_ragg()
+        }
+
+        USE_RAGG
+    }
+})
```

Comment:
I think this is simpler and easier to read?

```suggestion
use_ragg <- local({
    # Only check global option once per session
    delayedAssign("use_ragg", init_use_ragg())
    function() use_ragg
})
```